### ISSUE TITLE
[Android] Most Common Issues With RetroArch Android Version

### ISSUE THREAD

**Issue by thealidev** [libretro/RetroArch#15643]
- RetroArch on Android 11 and more not supported on google play store
![Screenshot_20230823-195030_Google_Play_Store](https://github.com/libretro/RetroArch/assets/84855696/b9cdfc4c-95a9-4d9b-a418-473af586e612)
- RetroArch Android doesn't have mic and midi drivers
![Screenshot_20230823-200107_RetroArch](https://github.com/libretro/RetroArch/assets/84855696/a3127c2a-abc8-45e3-bb71-454af0b0139f)
![Screenshot_20230823-200056_RetroArch](https://github.com/libretro/RetroArch/assets/84855696/6af9c86b-ef4b-422a-9d15-d0e5d0a0bec9)
- RetroArch Android sucks and force stops on many cores ( i can't take sc of issue)
- RetroArch Android comes with many bugs (lol)

**Comment by sonninnos** [comment]
Just use the version from RA website and not the Play Store version.

**Comment by thealidev** [comment]
i am using website version
i want mic support because i am playing `mario and Luigi Bowser's inside story` , so i need mic
how to enable it
plz it will be helpful

**Comment by hizzlekizzle** [comment]
There is no mic support in the core(s) currently beyond the press-button-for-noise implementation. There is a WIP melonDS update that includes actual mic support but no ETA on when it will be available from the online updater.

**Comment by thealidev** [comment]
mm
but their is really no possibly to use mic
and i am using `DeSuME` as a core

**Comment by hizzlekizzle** [comment]
Correct. RetroArch had no mic driver *at all* until very recently and that recent addition has not been hooked up in any widely available cores yet. The upcoming melonDS will be the first example.

**Comment by thealidev** [comment]
oh ok
but pc version have mic
